src/index.html

```diff
@@ -16,7 +19,183 @@
+                <img src="./images/beasound-theatre.png"
+                alt="beasound-theatre"
+                class="product__photo">
+
+                <h3 class="product__title">BEOSOUND THEATRE</h3>
+                <p class="product__category">Four patent-pending acoustic innovations in one sleek soundbar envelop you in sound from every angle. Made to fit any TV, and last for generations.</p>
+                <p class="product__price">$ 3199.00</p>
+              </article>
+            </div>
+          </section>
+
+          <section class="categories" id="categories">
+            <h2 class="section-title">Sound. Vision. <br>
+              It’s your pick.
+            </h2>
+
+            <div class="categories__items">
+              <article class="items">
+                <img src="images/gallery_speaker.png"
+                alt="gallery_speaker"
+                class="items__photo">
+
+                <img src="images/gallery_auto-falante.png"
+                alt="gallery_auto-falante"
+                class="items__photo">
+
+                <div>
+                  <h3 class="items__title">Portable speakers</h3>
+                </div>
+              </article>
+              <article class="items">
+                <img src="images/gallery_airbuds.png"
+                alt="gallery_airbuds"
+                class="items__photo">
+
+                <img src="images/gallery_airbuds-hr.png"
+                alt="gallery_airbuds-hr"
+                class="items__photo">
+
+                <div>
+                  <h3 class="items__title">Earphones</h3>
+                </div>
+              </article>
+              <article class="items">
+                <img src="images/gallery_caixa_circular.png"
+                alt="gallery_caixa_circular"
+                class="items__photo">
+
+                <img src="images/gallery_caixa-achatada.png"
+                alt="gallery_caixa-achatada"
+                class="items__photo">
+
+                <div>
+                  <h3 class="items__title">Smart Home</h3>
+                </div>
+              </article>
+            </div>
+          </section>
+
+          <section class="about-us" id="about-us">
+            <article>
+              <img src="images/background_footer.png" alt="background_footer">
+            </article>
+
+            <div class="about-us__content">
+              <h2 class="about-us__title">Timeless, for 50 <br>
+                years and counting
+              </h2>
+              <p class="about-us__description">
+                Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+              </p>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title">Contact us</h2>
```

Comment:
Here you should implement the form for contact us here. Call me on the chat if you need help with it.